client/src/dataProviders/publiclyDataProviders.js

```diff
@@ -22,3 +26,32 @@ export const entitiesProvider = (entitiesUrl: string) => {
     needed: false,
   }
 }
+
+export const entitiesSearchResultEidsProvider = (searchFor: string) => {
+  return {
+    ref: searchFor,
+    getData: [
+      fetch,
+      `${process.env.REACT_APP_API_URL || ''}/api/v/searchEntity?text=${searchFor}`,
+      {
+        accept: 'application/json',
+      },
+    ],
+    onData: [dispatchSearchEids],
```

Comment:
if you store in the format of object map, you can use the default dispatchReceivedData and don't need dispatchSearchEids